src/pages/profiles/ProfileDetailPanel.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { FC } from "react";
+import { Button, Icon } from "@canonical/react-components";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import Loader from "components/Loader";
+import Aside from "components/Aside";
+import usePanelParams from "util/usePanelParams";
+import { useNotify } from "context/notify";
+import { fetchProfile } from "api/profiles";
+import ProfileLink from "./ProfileLink";
+import { fetchProject } from "api/projects";
+import { isProjectWithProfiles } from "util/projects";
+import ExpandableList from "components/ExpandableList";
+import { isDiskDevice, isNicDevice } from "util/devices";
+import { getProfileInstances } from "util/usedBy";
+import ProfileUsedByProject from "./ProfileUsedByProject";
+
+const ProfileDetailPanel: FC = () => {
+  const notify = useNotify();
+  const panelParams = usePanelParams();
+
+  const profileName = panelParams.profile;
+  const projectName = panelParams.project;
+
+  const {
+    data: profile,
+    error,
+    isLoading: isProfileLoading,
+  } = useQuery({
+    queryKey: [queryKeys.profiles, profileName, projectName],
+    queryFn: () => fetchProfile(profileName ?? "", projectName),
+    enabled: profileName !== null,
+  });
+
+  const {
+    data: project,
+    error: projectError,
+    isLoading: isProjectLoading,
+  } = useQuery({
+    queryKey: [queryKeys.projects, projectName],
+    queryFn: () => fetchProject(projectName),
+  });
+
+  if (error) {
+    notify.failure("Loading profile failed", error);
+  }
+
+  if (projectError) {
+    notify.failure("Loading project failed", error);
+  }
+  const isLoading = isProfileLoading || isProjectLoading;
+
+  const featuresProfiles = isProjectWithProfiles(project);
+
+  const isDefaultProject = projectName === "default";
+  const usageCount = getProfileInstances(
+    projectName,
+    isDefaultProject,
+    profile?.used_by
+  ).length;
+
+  return (
+    <Aside width="narrow" pinned className="u-hide--medium u-hide--small">
```

Comment:
The skeleton and content for this panel is pretty much the same as the instance detail panel. We should extract a common component and reuse the CSS and structure to avoid duplication of code.